build/mixer.sh

```diff
@@ -342,8 +342,8 @@ if ! "${IS_DOWNSTREAM}"; then
     mixer_cmd repo remove "clear"
 fi
 
-mixer_cmd config set Swupd.CONTENTURL "${DISTRO_URL}/update"
-mixer_cmd config set Swupd.VERSIONURL "${DISTRO_URL}/update"
+mixer_cmd config set Swupd.CONTENTURL "${SWUPD_CONTENT_URL:-"${DISTRO_URL}/update"}"
```

Comment:
I think you can drop the "SWUPD" prefix.

I would also add CONTENT_URL and VERSION_URL to global.sh using the same approach we did for other vars:
```
CONTENT_URL=${CONTENT_URL:-${DISTRO_URL}/update}
```
Instead of having the fallback default being defined here. So the code looks more clean